src/main/java/Alvis.java

```diff
@@ -0,0 +1,129 @@
+                System.exit(0);
+                break;
+            case "list":
+                System.out.println("Let me help you to remember what you need to do ...");
+                for (int i = 0; i < taskList.size(); ++i) {
+                    System.out.println("" + (i + 1) + ". " + taskList.get(i).toString());
+                }
+                break;
+            case "toggle":
+                try {
+                    int index = Integer.parseInt(userInput.substring(7)) - 1;
+                    taskList.get(index).toggleStatus();
+                    System.out.println("As requested, I have changed the status of your task");
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("I cannot do that, the index is invalid");
+                }
+                writeToFile(taskList);
+                break;
+            case "todo":
+                try {
+                    Task toDo = new ToDo(userInput.substring(5));
+                    addTask(toDo, taskList);
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println("Please input a body for the task");
+                }
+                writeToFile(taskList);
+                break;
+            case "delete":
+                try {
+                    int index = Integer.parseInt(userInput.substring(7)) - 1;
+                    taskList.remove(index);
+                    System.out.println("As requested, I have deleted the task");
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("I cannot do that, the index is invalid");
+                }
+                writeToFile(taskList);
+                break;
+            case "deadline":
+                int byIndex = userInput.indexOf("/by");
+                Task deadline = new Deadline(userInput.substring(9, byIndex - 1), userInput.substring(byIndex + 4));
+                addTask(deadline, taskList);
+                writeToFile(taskList);
+                break;
+            case "event":
+                int fromIndex = userInput.indexOf("/from");
+                int toIndex = userInput.indexOf("/to");
+                Task event = new Event(userInput.substring(6, fromIndex - 1), userInput.substring(fromIndex + 6, toIndex - 1),
+                        userInput.substring(toIndex + 4));
+                addTask(event, taskList);
+                writeToFile(taskList);
+                break;
+            default:
+                System.out.println("Refer to the webpage for the commands");
+            }
+        }
+    }
+
+    public static void addTask(Task task, List<Task> taskList) {
+        taskList.add(task);
+        System.out.println("Understood, I have added to your list:\n" + task.toString());
+        System.out.println("You have " + taskList.size() + " tasks");
+    }
+
+    private static void loadTasks(List<Task> taskList) throws IOException {
```

Comment:
Idk if you have reached to that Level in the ip Progress where you'll have to implement better OOP. 
But can separate saving and loading tasks into a different class. 🐱 🐈